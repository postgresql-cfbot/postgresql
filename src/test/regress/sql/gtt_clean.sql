

reset search_path;

select pg_sleep(5);

drop schema gtt cascade;

