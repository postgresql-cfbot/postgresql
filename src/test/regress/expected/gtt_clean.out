reset search_path;
select pg_sleep(5);
 pg_sleep 
----------
 
(1 row)

drop schema gtt cascade;
NOTICE:  drop cascades to 5 other objects
DETAIL:  drop cascades to table gtt.gtt1
drop cascades to table gtt.gtt2
drop cascades to table gtt.gtt3
drop cascades to table gtt.gtt_t_kenyon
drop cascades to table gtt.gtt_with_seq
